pydantic_ai_slim/pydantic_ai/profiles/zhipu.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations as _annotations
+
+from .openai import OpenAIModelProfile
+
+
+def zhipu_model_profile(model_name: str) -> OpenAIModelProfile:
+    """Get the model profile for a Zhipu AI model.
+
+    Zhipu AI provides OpenAI-compatible API, so we use OpenAIModelProfile.
+
+    Args:
+        model_name: The Zhipu model name (e.g., 'glm-4.6', 'glm-4.5', 'glm-4.5-air', 'glm-4.5v').
+
+    Returns:
+        Model profile with Zhipu-specific configurations.
+    """
+    # Vision models — docs show vision variants with a trailing `v` like `glm-4.5v`
+    # Ref: https://docs.bigmodel.cn/cn/guide/develop/openai/introduction
+    is_vision_model = model_name.startswith(('glm-4.5v', 'glm-4v')) or (
+        'v' in model_name and ('glm-4.5v' in model_name or 'glm-4v' in model_name)
+    )
```

Comment:
This check is very redundant: `'v' in model_name` and `model_name.startswith('glm-4.5v')` are both subsets of `'glm-4.5v' in model_name`. Should we just check for the `v`?